doc/index.rst

```diff
@@ -161,3 +161,16 @@ Strawberry Fields is **free** and **open source**, released under the Apache Lic
    code/backend.gaussian
    code/backend.fock
    code/backend.tf
+
+
+.. toctree::
+   :maxdepth: 2
+   :caption: Applications Layer
+   :hidden:
+
+   code/apps/sample
+   code/apps/graph.dense
+   code/apps/graph.resize
+   code/apps/graph.sample
+   code/apps/graph.utils
+   code/apps/graph.max_clique
```

Comment:
It probably makes more sense to have this before the backend section (since users will not interact with backends directly). Unless this was deliberate to keep the apps layer 'out of sight'?

Reply:
Yes I was thinking to keep things on the down low for now and when the docs refactor is done we can think about where to place it and adding some introductory text, does that work?

Reply:
I'm happy with it for now!